Run actions documentation  added (#7448)
opencv/cvat: site/content/en/docs/enterprise/shapes-converter.md

---
title: 'Shapes converter'
linkTitle: 'Shapes converter'
weight: 4
description: 'How to perform bulk actions on filtered shapes'
---

The shapes converter is a feature that enables bulk actions on filtered shapes. For example, it allows you to delete

> all filtered shapes or convert masks to polygons, among other functions.

Reviewer: From this sentence I may conclude that shape converter allows me to delete filtered shapes

There is a difference between Shape converters and Annotations actions, that is not the same thing:

- Shape converters are subset of annotations actions, available for enteprise
- Annotation actions themselfes is open source feature

> > **Note:** That currently, shape conversion is only applicable to **Shapes** and cannot be used for **Tracks**.

Reviewer: That is about annotation actions, they allow to modify only shapes

See:

- [Run actions menu](#run-actions-menu)
- [Convert shapes](#convert-shapes)

## Run actions menu

> The Shapes Converter can be accessed from the annotation menu. To access it, click on the burger icon

Reviewer: ```suggestion
Annotations actions can be accessed from the annotation menu. To access it, click on the burger icon
```

---
title: 'Shapes converter'
linkTitle: 'Shapes converter'
weight: 4
description: 'How to perform bulk actions on filtered shapes'
---

> The shapes converter is a feature that enables bulk actions on filtered shapes. For example, it allows you to convert
> masks to polygons, rectangle to polygons and other functions.

Reviewer: ```suggestion
The shapes converter is a feature that enables bulk actions on filtered **shapes**. It allows you to perform mutual conversion between masks, polygons and rectangles.
```

The shapes converter functionality also includes **Remove filtered shapes** functionality, that is

> available for all plans without limitations.

Reviewer: Please, remove that. 

Remove filtered shapes is not a shapes converter, it is annotations action.

Author: It is in the same interface I cannot throw it away because it is a description of the window

Reviewer: Then we need annotations action section, because again, "remove filtered shapes" is not a shape converter.

Author: ok ¯\_(ツ)_/¯

| ---------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | **Select action**                        | Drop-down list with available actions: <br><li>**Remove filtered shapes** - removes all shapes in alignment with the set-up filter. Doesn't work with tracks.</li><li>**Shapes converter: masks to polygons** - converts all masks to polygons in alignment with the set-up filter.</li><li>**Shapes converter: masks to rectangles** - converts all masks to rectangles in alignment with the set-up filter.</li><li>**Shapes converter: polygon to masks** - converts all polygons to masks in alignment with the set-up filter.</li><li>**Shapes converter: polygon to rectangles** - converts all polygons to rectangles in alignment with the set-up filter.</li><li>**Shapes converter: rectangles to masks** - converts all rectangles to masks in alignment with the set-up filter.</li><li>**Shapes converter: rectangles to polygons** - converts all rectangles to polygons in alignment with the set-up filter.</li><br>**Note:** only **Remove filtered shapes** is available on the **Free** plan. |

Reviewer: Do you think we need to mention anytime `with the set-up filter`?
